df.creature-raws.xml

```diff
@@ -828,7 +901,7 @@
             <int16_t name='clutch_size_max'/>
             <int16_t name='vision_arc_min'/>
             <int16_t name='vision_arc_max'/>
-            <int32_t name='speed' comment='no longer used'/>
+            <int32_t name='unused_01'/> bay12: int32_t[CreatureCasteLong]
```

Comment:
The Units module was still looking at this field, even though it was removed many years ago when Gaits were added

Reply:
how should the logic of Units::computeMovementSpeed change?